### PR TITLE
Add tree-grid-directive@0.4.0 w/ git auto-update

### PR DESCRIPTION
**Pull request by sufuf3**
Pull request for issue: #9478
Related issue(s): #10141 #11170

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/khan4019/tree-grid-directive
 * Official website (optional, not the repository): https://khan4019.github.io/tree-grid-directive/test/treeGrid.html
 * NPM package url (optional): https://www.npmjs.com/package/angular-bootstrap-grid-tree
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 38
   - Count of stars: 397
   - Count of forks: 170
 * NPM download stats (optional):
   - Downloads in the last day: 7
   - Downloads in the last week: 70
   - Downloads in the last month: 279

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sufuf3** [comment]
@cdnjs/intern3 
Please help me review this PR, Thank you.

Although the library can use npm install (https://github.com/khan4019/tree-grid-directive#mininum-to-start), I still think we should get from git because the npm Collaborators is not the GitHub author. And the files from npm is not as the same with from GitHub. (use md5sum)

File: ajax/libs/angular-bootstrap-grid-tree/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "angular-bootstrap-grid-tree",
```
Comment (extend1994) [inline ajax/libs/angular-bootstrap-grid-tree/package.json]: Let us use "tree-grid-directive" which is the same as repo name. Also do not forget to revise folder name and PR title.

**Comment by sufuf3** [comment]
@extend1994 I change to `tree-grid-directive` 
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
ping @cdnjs/intern3

**Comment by sufuf3** [comment]
@cdnjs/library-reviewer 
Please help me review this PR, Thank you.

**Pull request merged**